aliases/available/general.aliases.bash

```diff
@@ -18,6 +18,11 @@ alias l1='ls -1'
 
 alias _="sudo"
 
+# Shortcuts to edit startup files
+alias vigen="vim ~/.bash_it/aliases/available/general.aliases.bash"
+alias vbrc="vim ~/.bashrc"
```

Comment:
I do actually think it makes sense to have options for both.  I don't use these files interchangeably, and rather prefer a hierarchical separation in how I organize these files (I'm perhaps making this sound more sophisticated than it is, -the uncomfortable reality is that I actually do far more in .bashrc than I probably should, including some lengthier functions, but do try to specifically limit how much I have in my .bash_profile for the sake of simplicity).